components/look-at/index.js

```diff
@@ -39,6 +39,7 @@ AFRAME.registerComponent('look-at', {
   init: function () {
     this.target3D = null;
     this.vector = new THREE.Vector3();
+    this.isCamera = this.el.getObject3D('camera') ? true : false;
```

Comment:
You could add an event listener for componentinitialized, but it adds asynchrony. The check is very cheap since it's accessing a map and seeing its truthiness